### PR TITLE
Add picamera support

### PR DESCRIPTION
**Pull request by bhargavthriler**
python3-picamera : recipe for python picamera library.
python3-colorzero: recipe for python colorzero library, required by picamera
picamera-libs    : recipe raspberry pi firmware which are required by picamera.

Closes: #959
Signed-off-by: bhargavthriler <bhargavthriler@gmail.com>

**- What I did**
Created recipe for python picamera library. Along with it created 2 more, python colorzero and picamera-libs.

### CODE REVIEW

File: recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+SUMMARY = "colorzero is a color manipulation library for Python"
```
Comment (agherzan) [inline recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb:1]: Can we move this to meta-python?

**Review by agherzan — CHANGES_REQUESTED** [review]

File: recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb (hunk 1 above)
Comment (bhargavthriler) [reply recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb:1]: By meta-python you mean inside openembedded.org/meta-openembedded/meta-python ?

Comment (agherzan) [reply recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb:1]: Yes. That is correct. http://layers.openembedded.org/layerindex/branch/master/layer/meta-python/

Comment (bhargavthriler) [reply recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb:1]: @agherzan  I have sent a patch to meta-python, let's wait till it is merged. Than I will push a new commit here.

Comment (agherzan) [reply recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb:1]: Thanks!

Comment (bhargavthriler) [reply recipes-multimedia/python3-colorzero/python3-colorzero_2.0.bb]: @agherzan I have made the change, the python3-colorzero recipe is merged. 
http://cgit.openembedded.org/meta-openembedded/tree/meta-python/recipes-devtools/python/python3-colorzero_2.0.bb

**Review by agherzan — CHANGES_REQUESTED** [review]
Let's clean up the MR by dropping the upstreamed recipe completely from it. No need to add and remove in the same MR.

**Comment by bhargavthriler** [comment]
> Let's clean up the MR by dropping the upstreamed recipe completely from it. No need to add and remove in the same MR.

You mean create a new PR which doesn't have reference to upstream recipe and delete this ?

**Comment by agherzan** [comment]
@bhargavthriler no, just force push to you branch rebasing on top of master and cleaning up the changes.

**Comment by bhargavthriler** [comment]
> @bhargavthriler no, just force push to you branch rebasing on top of master and cleaning up the changes.

@agherzan made the changes, please check

**Review by agherzan — APPROVED** [review]

**Pull request merged**